Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -509,10 +509,12 @@ void MediaPlayerPrivateGStreamer::pause()
     if (currentState < GST_STATE_PAUSED && pendingState <= GST_STATE_PAUSED)
         return;
 
-    if (changePipelineState(GST_STATE_PAUSED))
+    if (changePipelineState(GST_STATE_PAUSED)) {
+        m_paused = true;
         GST_INFO("Pause");
-    else
+    } else {
         loadingFailed(MediaPlayer::Empty);
+    }
```

Comment:
Remove this closing brace and also the opening one. Single lines don't need braces, according to the WebKit code style guidelines.